src/presentation/ui/component/structural/deleteModal.templ

```diff
@@ -31,9 +31,11 @@ templ DeleteModal(
 					x-text={ elNameStatePath }
 					class="rounded-md border-2 border-dashed border-red-900 px-2 py-1"
 				></strong>
-				<span x-show={ elIdStatePath + " !== ''" }>
-					#<strong x-text={ elIdStatePath }></strong>
-				</span>?
+				if elIdStatePath != "" {
```

Comment:
Interesting. Why use Go AND Alpine for this? Aren't you checking the same thing twice?

Reply:
We discussed this issue during the call. We agreed to remove `x-show` and let Go handle the content.